packages/api/tests/app/test-launchpadAuth.js

```diff
@@ -144,3 +144,27 @@ test.serial('Request returns an unauthorized response when validation response d
   t.is(response.body.message, 'User not authorized');
   stub.restore();
 });
+
+test.serial('Requests to /token endpoint are disabled.', async (t) => {
+  const response = await request(app)
+    .get('/token')
+    .set('Accept', 'application/json')
+    .expect(501);
+
+  t.regex(response.body.message, /Login with launchpad/);
+});
+
+test.serial.only('Non-Launchpad protected API explicitly disallows valid Launchpad tokens.', async(t) => {
```

Comment:
You need to remove `.only` from this test

Reply:
D'oh! I thought I had a gitsecret to prevent that from happening again.  👍